<ui version="4.0" >
 <class>Kleo::NewCertificateUi::OverviewPage</class>
 <widget class="QWizardPage" name="Kleo::NewCertificateUi::OverviewPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>147</height>
   </rect>
  </property>
  <property name="title" >
   <string comment="@title" >Review Certificate Parameters</string>
  </property>
  <property name="subTitle" >
   <string>Please review the certificate parameters before proceeding to certificate </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QTextBrowser" name="textBrowser" >
     <property name="openLinks" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showAllDetailsCB" >
     <property name="text" >
      <string>Show all details</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
